src/mlpack/methods/reinforcement_learning/environment/ftn.hpp

```diff
@@ -0,0 +1,300 @@
+/**
+ * @file methods/reinforcement_learning/environment/ftn.hpp
+ * @author Nanubala Gnana Sai
+ *
+ * This file is an implementation of Fruit Tree Navigation (FTN) Task:
+ * https://github.com/RunzheYang/MORL/blob/master/synthetic/envs/fruit_tree.py
```

Comment:
Additionally we should use the citation mentioned in the README.md - https://github.com/RunzheYang/MORL/#citation.